apps/gitness/src/pages/repo/repo-branch-list.tsx

```diff
@@ -15,16 +14,25 @@ import {
   Text
 } from '@harnessio/canary'
 import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
+import { usePagination } from '../../framework/hooks/usePagination'
+
 import { useListBranchesQuery, TypesBranch } from '@harnessio/code-service-client'
 import { timeAgo } from '../pipeline-edit/utils/time-utils'
 
 const filterOptions = [{ name: 'Filter option 1' }, { name: 'Filter option 2' }, { name: 'Filter option 3' }]
 const sortOptions = [{ name: 'Sort option 1' }, { name: 'Sort option 2' }, { name: 'Sort option 3' }]
 
 export function ReposBranchesListPage() {
+  // lack of data: total branches
+  // hardcoded
+  const totalPages = 10
```

Comment:
can we just add this to a constant file and just export it

Reply:
Thats a good point. It'll reduce the changes once we get the API working. 

 Will change in the next PR later today.

Reply:
thanks @sans-harness